src/oled_display_node.cpp

```diff
@@ -136,8 +137,10 @@ int main(int argc, char** argv) {
 #include <sensor_msgs/BatteryState.h>
 
 // Some limited state for display
+std::string g_batteryPercentage = "    ";
+std::string g_firmwareVersion = "---";
```

Comment:
For Kinetic images we are likely going to have to do an entirely different ubiquity_motor PR to pull in what you did for noetic ubiquity_motor.  To me it seems acceptable that firmware rev is an undone feature at this time.